src/components/new-request/business-lookup.vue

```diff
@@ -34,7 +34,7 @@
 
       <template v-slot:no-data>
         <p class="pl-5 font-weight-bold">
-          No active B.C. business found
+          {{  lookupNoActiveText  }}
```

Comment:
You don't need double spaces in here. (Actually, you don't need any spaces, but our code style is to include one leading and one trailing space.)